eZ/Publish/Core/Repository/Values/ContentType/FieldDefinition.php

```diff
@@ -116,7 +116,7 @@ public function getDescriptions()
      */
     public function getDescription($languageCode)
     {
-        if (array_key_exists($languageCode, $this->descriptions)) {
+        if (is_array($this->descriptions) && array_key_exists($languageCode, $this->descriptions)) {
```

Comment:
I'd also prefer if the property was made an array, since it is documented as such.

Check https://github.com/ezsystems/ezpublish-kernel/blob/e2d26faa54715675fc8f1eef3defbe192973bf5b/eZ/Publish/Core/Persistence/Legacy/Content/Type/Mapper.php#L186-L186, this is where FieldDefinition objects are initialized. This builds the Persistence (SPI) object, which is then mapped to API objects [there](https://github.com/ezsystems/ezpublish-kernel/blob/c30f04e9cd24cfd04ce6f5d614812e6527c87cd5/eZ/Publish/Core/Repository/Helper/ContentTypeDomainMapper.php#L227-L227).